src/chain_sync/chain_muxer.rs

```diff
@@ -512,6 +515,48 @@ where
         Ok(Some((tipset, source)))
     }
 
+    fn handle_network_messages(&self) -> ChainMuxerFuture<(), ChainMuxerError> {
```

Comment:
`handle_network_messages` is fairly generic. This is the main event loop for the stateless mode, right?

```suggestion
    fn stateless_node(&self) -> ChainMuxerFuture<(), ChainMuxerError> {
```